include/Prefs.h

```diff
@@ -399,6 +399,7 @@ class Prefs {
   inline bool do_snmp_polling() { return (snmp_polling); };
   inline bool do_active_monitoring(){ return (active_monitoring); };
   inline bool do_tls_quic_hostnaming() { return (tls_quic_hostnaming); };
+  inline bool get_starttls(){ return (starttls); };
```

Comment:
I suggest to rename this method to something more meaningful like "enable_email_starttls"